client/src/modules/stock/aggregated_consumption/templates/days_stock_out.tmpl.html

```diff
@@ -0,0 +1,12 @@
+<div class="ui-grid-cell-contents">
+  <input
+    ng-if="row.groupHeader"
+    name="days_stock_out"
```

Comment:
Let's make this default to "0".

Reply:
I wanted the user to write really zero instead of using a default value, this will prevent him from making the mistake of forgetting